app/views/main.hbs

```diff
@@ -51,6 +51,17 @@
       <meta name="monetization" content="{{.}}">
     {{/config.monetization}}
 
+    <script>
+      if (document.monetization) {
+        document.monetizationExtensionInstalled = true
+      } else {
+        document.monetization = document.createElement('div')
+        document.monetization.state = 'stopped'
+      }
+    </script>
+
```

Comment:
Do we still need any of this? Since it's in the bundle already.

Reply:
For some reason not including it seems to make the app hang atm..

Reply:
In fact, I can't get get the monetization script to initialize without including it in the head tag